.github/workflows/job_python_unit_tests.yml

```diff
@@ -231,7 +234,7 @@ jobs:
           PYTORCH_TRACING_MODE: TORCHFX
 
       - name: ONNX Layer Tests
-        if: fromJSON(inputs.affected-components).ONNX_FE.test
+        if: ${{ fromJSON(inputs.affected-components).ONNX_FE.test && inputs.python-version != '3.12' }} # Ticket: 152240
```

Comment:
looks like it's available https://pypi.org/project/onnxruntime/#files, but only for glibc 2.27 and newer

looks like we need to change the condition

Reply:
it seems like that it is installed in the Layer tests dependencies step: https://github.com/openvinotoolkit/openvino/actions/runs/10812194091/job/29994666552#step:16:117 but is not found during tests execution.